File: Backend-Java/src/main/java/org/example/DAL/constants/Roles.java
package org.example.DAL.constants;

import java.util.*;

public class Roles {
    public static String Admin="Admin";
    public static String Moder="Moder";
    public static String Editor="Editor";
    public static String Supporter="Supporter";
    public static String PermanentUser="Permanent User";
    public static String User="User";
    public static final List<String> All = Arrays.asList(
            Admin,
            Moder,
            Editor,
            Supporter,
            PermanentUser,
            User
    );

}

File: Backend-C#/Data.MyShop/Constants/Roles.cs
namespace Data.MyShop.Constants
{
    public static class Roles
    {
        public static List<string> All = new()
        {
            Admin,
            Moder,
            Editor,
            Supporter,
            PermanentUser,
            User,
        };

        public const string Admin = "Admin";
        public const string Moder = "Moder";
        public const string Editor = "Editor";
        public const string Supporter = "Supporter";

        public const string PermanentUser = "Permanent User";
        public const string User = "User";
    }
}
